### PR TITLE
ES : fix configuration

### PR DESCRIPTION
**Pull request by nplanel**
ES specific configuration need to be keep in storage.elasticsearch section

### CODE REVIEW

**Comment by nplanel** [comment]
run skydive-unit-tests

**Comment by lebauce** [comment]
@nplanel Can you please rebase ?

File: etc/skydive.yml.default [hunk 1]
```diff
@@ -75,17 +75,6 @@ analyzer:
     # Max number of flows in write buffer (after which all flows accumulated are dropped)
     # max_flow_buffer_size: 100000
 
-    # If a limit is specified, when the index reaches it, it is rolled.
```
Comment (lebauce) [inline etc/skydive.yml.default]: Those parameters still exist, they just need to moved to the right section

Comment (nplanel) [reply etc/skydive.yml.default]: yes they exist under storage.elastricsearch or storage.newbackend.driver = elasticsearch
but should not exist under analyzer.flow

**Pull request merged**